lib/cli/run-option-metadata.js

```diff
@@ -114,3 +105,23 @@ const ALL_MOCHA_FLAGS = Object.keys(TYPES).reduce((acc, key) => {
 exports.isMochaFlag = flag => {
   return ALL_MOCHA_FLAGS.has(flag.replace(/^--?/, ''));
 };
+
+/**
+ * Returns expected yarg option type for a given mocha flag.
+ * @param {string} flag - Flag to check (can be with out without leading "--"")
+ * @returns {string | undefined} - If flag is a valid mocha flag, the expected type of argument for this flag is returned, otherwise undefined is returned.
+ */
+exports.expectedTypeForFlag = flag => {
+  const normalizedName = flag?.replace(/^--?/, '');
```

Comment:
> `flag?.`

[Refactor] `flag` is marked as `@param {string}`, not `@param {string | undefined}`. So in theory this `?.` can be replaced with a `.`:

```suggestion
  const normalizedName = flag.replace(/^--?/, '');
```

I tried it out locally and no new tests failed. So I think this is a good replacement to make? But if it isn't, then the JSDoc type should be switched to `string | undefined`.

Reply:
Removing the `?` since it's not needed.